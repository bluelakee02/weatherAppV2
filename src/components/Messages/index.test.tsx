import * as React from 'react';
import {shallow, mount, render} from 'enzyme';

import { Loading } from './index';

describe('Messages test', () => {
  it('should match the snapshot', () => {
    expect(shallow(<Loading/>)).toMatchSnapshot();
  });


  it('should mount in a full DOM', () => {
    expect(mount(<Loading/>).find('div').length).toBe(1);
  });

  it('should render to static HTML', () => {
    expect(render(<Loading/>).text()).toEqual('Loading...');
  });
});
