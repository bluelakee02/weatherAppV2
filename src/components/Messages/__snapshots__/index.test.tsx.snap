// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages test should match the snapshot 1`] = `
<div>
  Loading...
</div>
`;
